### DISCUSSION TITLE
update info

### DISCUSSION THREAD

**Discussion by paulpla** [brianpetro/obsidian-smart-connections#682] [General]
is there any info about what's new on every update (I search and did not find)..
Because for every update the plug in start to embed again, and it takes a long time in my machine.. is this normal?

**Comment by brianpetro** [comment]
Hi @paulpla 

Brief release notes can be found here https://github.com/brianpetro/obsidian-smart-connections/releases

I try to make note of releases that would trigger re-embeddings, but sometimes the change is related to a bug fix or other change that I didn't expect to cause a re-embedding.

There was a recent change that might disproportionately impact re-embeddings on windows computers (normalizing the newline character). But since this change happened in the underlying [Smart Chunks](https://github.com/brianpetro/jsbrains/tree/main/smart-chunks#readme) project, there wasn't an explicit note in the plugin release about the potential for re-embeddings.

Overall, full re-embeddings should be uncommon, even after an update. A good way to test if it is a bug or caused by an update:

After the embeddings are done processing, go to the community plugins section, disable, then re-enable Smart Connections. If the embedding process starts again, then it is likely a bug and should be reported as an issue here on GitHub 🌴
